rollup.config.js

```diff
@@ -1,158 +0,0 @@
-/* eslint-disable sort-keys */
-import babel from '@rollup/plugin-babel';
-import commonjs from '@rollup/plugin-commonjs';
-import image from '@rollup/plugin-image';
-import json from '@rollup/plugin-json';
-import resolve from '@rollup/plugin-node-resolve';
-import typescript from '@rollup/plugin-typescript';
-import external from 'rollup-plugin-peer-deps-external';
-import url from 'rollup-plugin-url';
-import copy from 'rollup-plugin-copy';
-import globals from 'rollup-plugin-node-globals';
-import visualizer from 'rollup-plugin-visualizer';
-import replace from '@rollup/plugin-replace';
-import builtins from '@stream-io/rollup-plugin-node-builtins';
-import { terser } from 'rollup-plugin-terser';
-import { prepend } from 'rollup-plugin-insert';
-import process from 'process';
-import pkg from './package.json';
-
-process.env.NODE_ENV = 'production';
-
-const baseConfig = {
-  cache: false,
-  input: {
-    index: 'src/index.ts',
-    'components/Emojis/index': 'src/components/Emojis/index.ts',
-  },
-  watch: {
-    chokidar: false,
-  },
-};
-
-const externalDependencies = [
-  /@babel/,
-  '@braintree/sanitize-url',
-  '@fortawesome/free-regular-svg-icons',
-  '@fortawesome/react-fontawesome',
-  '@juggle/resize-observer',
-  '@stream-io/transliterate',
-  'custom-event',
-  /dayjs/,
-  /emoji-mart/,
-  'emoji-regex',
-  'i18next',
-  'isomorphic-ws',
-  'linkifyjs',
-  'lodash.debounce',
-  'lodash.isequal',
-  'lodash.throttle',
-  'lodash.uniqby',
-  'mml-react',
-  'pretty-bytes',
-  'prop-types',
-  'react-fast-compare',
-  'react-images',
-  'react-image-gallery',
-  'react-is',
-  'react-player',
-  'react-textarea-autosize',
-  'react-virtuoso',
-  'textarea-caret',
-  /uuid/,
-];
-
-const basePlugins = ({ useBrowserResolve = false }) => [
-  replace({
-    preventAssignment: true,
-    'process.env.NODE_ENV': JSON.stringify('production'),
-  }),
```

Comment:
Are we missing this in the esbuild pipeline?

Reply:
I think that is being set by default when building with `esbuild`:
https://esbuild.github.io/api/#platform

Reply:
Nice! TIL :)